setup.py

```diff
@@ -1,5 +1,8 @@
 import setuptools
 
+# get the version number
+exec(open('src/pytom_tm/_version.py').read())
+
```

Comment:
I really dislike using `exec`, also [using `__version__` inside packages is a rejected PEP](https://peps.python.org/pep-0396/#pep-rejection).
[This answer](https://stackoverflow.com/a/73119016/21764481) would be more suitable, so hardcode the version in `setup.py` (this allows us to use configure files later on) and then using 
```python
from importlib.metadata import version
version = version('pytom_tm') #guessing the name here
``` 
as a version file or inside `src/pytom_tm/__init__.py`